docs/source/configuration/cors.mdx

```diff
@@ -35,8 +35,8 @@ server:
     # (Ignored if allow_any_origin is true)
     # (Defaults to the Apollo Studio url: `https://studio.apollographql.com`)
     origins:
-      - https://www.your-app.example.com/
-      - https://studio.apollographql.com/ # Keep this so Apollo Studio can run queries against your router
+      - https://www.your-app.example.com
```

Comment:
Since this caught us out and isn't really intuitive, maybe a comment at line 36.5 to explain not to include trailing slashes?

Reply:
That's a great idea!

Reply:
fixed in [c53c268](https://github.com/apollographql/router/pull/1471/commits/c53c26839e4d3c36c2da5798419a145b95b477f1)